tamr_toolbox/workflow/jobs.py

```diff
@@ -60,6 +63,21 @@ def run(
             LOGGER.error(error_msg)
             raise NotImplementedError(error_msg)
 
+        # Excluding GOLDEN_RECORDS as profile information is refreshed as part of job:
+        # breakpoint()
+        if (
+            (target_type == ProjectType.SCHEMA_MAPPING_RECOMMENDATIONS)
+            | (target_type == ProjectType.CATEGORIZATION)
+            | (target_type == ProjectType.DEDUP)
+        ):
+            if run_profile_unified_datasets:
+                try:
+                    profile = project.unified_dataset().profile()
```

Comment:
could you add some LOGGER.info('') statements here as well. For example 
LOGGER.info('Creating profile of unified dataset')
LOGGER.info('Updating profile of unified dataset')